common/lib/xmodule/xmodule/public/js/vertical_student_view.js

```diff
@@ -0,0 +1,16 @@
+/* JavaScript for Vertical Student View. */
+window.VerticalStudentView = function (runtime, element) {
+
+    RequireJS.require(['js/bookmarks/views/bookmark_button'], function (BookmarkButton) {
```

Comment:
@andy-armstrong - what's the status of requirejs in xmodules? I had previously thought that it wasn't available, but after checking the [RequireJS in the LMS](https://openedx.atlassian.net/wiki/display/TNL/RequireJS+in+the+LMS) page it looks like the Video Module has taken this approach. So the `xmodule.js` template provides requirejs to the window? I just want to verify that this will cooperate with our production pipeline.